libgalois/include/katana/PropertyGraph.h

```diff
@@ -262,38 +262,51 @@ class KATANA_EXPORT PropertyGraph {
     rdg_.set_mirror_nodes(std::move(a));
   }
 
+  /// Return the node property table for local nodes
+  const std::shared_ptr<arrow::Table>& node_properties() const {
+    return rdg_.node_properties();
+  }
+
+  /// Return the edge property table for local edges
+  const std::shared_ptr<arrow::Table>& edge_properties() const {
+    return rdg_.edge_properties();
+  }
+
```

Comment:
Do we want to expose this? Will we always be able to construct an arrow table regardless of the internal storage of the properties?

Reply:
I'm not following.  I moved these functions from public to private to stop exposing them.

Reply:
My bad. I wasn't paying attention. This may also invalidate my global comment.